.gitignore

```diff
@@ -41,6 +41,7 @@ htmlcov/
 .coverage
```

Comment:
> Unit test that pass tarantoolctl output to JSON serializer failed [4]
> with Python 3.5

This part of the commit message becomes a bit outdated and now it is more relevant to the second commit.

Reply:
updated
```
commit 73a78c309621bfc312dac3dd5e146c7e928b7267 (HEAD -> ligurio/add-unittest)
Author: Sergey Bronnikov <sergeyb@tarantool.org>
Date:   Fri Mar 26 10:45:12 2021 +0300

    test: add unit tests
    
    Tests written using unittest [1] and hypothesis [2] modules. We use
    hypothesis v4.57.0 because it is a latest version with Python 2.7
    support [3].
    
    How-to run:
    
    $ make test_unittest
    
    1. https://docs.python.org/3/library/unittest.html
    2. https://hypothesis.readthedocs.io/en/latest/
    3. https://hypothesis.readthedocs.io/en/latest/changes.html#v4-57-0

```

Reply:
(Forgot to push it?)

Reply:
Pushed.